src/server_manager/electron_app/http/fetch.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 The Outline Authors
```

Comment:
Let's move this file to `infrastructure/` as well (same reasoning as the other comment)

Reply:
Done